include/z64audio.h

```diff
@@ -104,20 +112,20 @@ typedef struct {
 typedef struct {
     /* 0x0 */ s16 delay;
     /* 0x2 */ s16 arg;
-} AdsrEnvelope; // size = 0x4
+} EnvelopePoint; // size = 0x4
 
 typedef struct {
     /* 0x00 */ u32 start;
     /* 0x04 */ u32 end;
     /* 0x08 */ u32 count;
-    /* 0x0C */ char unk_0C[0x4];
-    /* 0x10 */ s16 state[16]; // only exists if count != 0. 8-byte aligned
+    /* 0x0C */ u32 originalLength;
+    /* 0x10 */ s16 predictorState[16]; // The predictor output state for the first frame of the loop
 } AdpcmLoop; // size = 0x30 (or 0x10)
 
 typedef struct {
     /* 0x00 */ s32 order;
-    /* 0x04 */ s32 npredictors;
-    /* 0x08 */ s16 book[1]; // size 8 * order * npredictors. 8-byte aligned
+    /* 0x04 */ s32 numPredictors;
+    /* 0x08 */ s16 book[1]; // Predictor coefficients, where each book size is 8 * order * npredictors. 8-byte aligned
```

Comment:
```suggestion
    /* 0x08 */ s16 book[1]; // Predictor coefficients, where book is of variable array length, each book size is 8 * order * npredictors. 8-byte aligned
```
Perhaps to emphasize that the "1" in `book[1]` is just a placeholder for a variable size?